client-react/src/Components/ChatWindowContext.js

```diff
@@ -63,9 +63,7 @@ const ChatReducer = (state, action) => {
 			console.log('status', newState.status);
 			return newState;
 		case 'receive-msg':
-			break;
```

Comment:
How/what are we returning if we break here?

Reply:
Break terminates the switch-case. JS returns undefined if nothing is explicitly returned by the function.